src/azure-cli/azure/cli/command_modules/acr/repository.py

```diff
@@ -154,7 +154,7 @@ def acr_repository_show_tags(cmd,
         username=username,
         password=password,
         repository=repository,
-        permission='pull')
+        permission='metadataRead')
```

Comment:
Is there a permission enum in models that we can use instead of hardcoding. Else we end up with changes in multiple places like this case.